ReSharper.FSharp/src/FSharp.Psi.Features/src/LanguageService/FSharpElementFactory.fs

```diff
@@ -210,3 +210,17 @@ type FSharpElementFactory(languageService: IFSharpLanguageService, psiModule: IP
             addParensIfNeeded rightArg |> ignore
 
             expr
+
+        member x.CreateSetExpr(left: ISynExpr, right: ISynExpr) =
+            let source = "() <- ()"
+            let expr = getExpression source
+            let setExpr = expr.As<ISetExpr>()
```

Comment:
Please use `:?>` here since we are expecting the expression to be of this type and not trying to do a safe cast which may produce a null reference.